### PR TITLE
[PPML] add driver and executor mem to ppml-submit

### PR DESCRIPTION
**Pull request by Uxito-Ada**
As the title.

### 2. User API changes

Add driver-memory and executor-memory to bigdl-ppml-submit.

### 4. How to test?
- [ ] N/A
- [ ] Unit test
- [ ] Application test
- [ ] Document test
- [ ] ...

### 5. New dependencies

- [ ] New Python dependencies
       - Dependency1 
       - Dependency2
       - ...
- [ ] New Java/Scala dependencies and their license
       - Dependency1 and license1
       - Dependency2 and license2
       - ...

### CODE REVIEW

File: ppml/trusted-big-data-ml/python/docker-gramine/base/bigdl-ppml-submit.sh [hunk 1]
```diff
@@ -23,11 +23,21 @@ while [[ $# -gt 0 ]]; do
       shift # past argument
       shift # past value
       ;;
+    --driver-memory)
```
Comment (liu-shaojun) [inline ppml/trusted-big-data-ml/python/docker-gramine/base/bigdl-ppml-submit.sh:26]: I remember that the two parameters sgx-executor-memory and sgx-driver-memory were deleted before, why are they added back now?

Comment (Uxito-Ada) [reply ppml/trusted-big-data-ml/python/docker-gramine/base/bigdl-ppml-submit.sh:26]: We deleted sgx-memory ones before, as they had been not needed by gramine.
driver-memory and executor-memory are different ones that specify the general memory for spark pods.

There are 3 kinds of memory for spark k8s: general memory (this PR), jvm memory, and sgx memory (deleted).

Comment (liu-shaojun) [reply ppml/trusted-big-data-ml/python/docker-gramine/base/bigdl-ppml-submit.sh:26]: ok I see, thanks for your explanation. Could you also update ppml/README.md to add these two new parameters?

Comment (Uxito-Ada) [reply ppml/trusted-big-data-ml/python/docker-gramine/base/bigdl-ppml-submit.sh:26]: sure

**Pull request merged**